app/controllers/photos_controller.rb

```diff
@@ -51,6 +56,7 @@ def update
 
   # DELETE /photos/1 or /photos/1.json
   def destroy
+    authorize @photo
```

Comment:
maybe you can create a private method for authorizing and use before_action so you can rid the repeated code

Reply:
^ hell ye